app/controllers/decidim/anonymous_codes/admin/codes_controller.rb

```diff
@@ -64,6 +63,12 @@ def export
 
         private
 
+        def base_query
+          query = tokens
+          query = query.ransack(params[:q]).result if params[:q].present?
+          query.order(:token)
```

Comment:
```suggestion
          tokens
```
this should suffice!

Reply:
Done